app/views/pages/profile.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="container mt-5">
+  <!-- Content here -->
+  <div class="d-flex flex-row bd-highlight mb-3 justify-content-between">
+    <div class="d-flex">
+      <img src="https://source.unsplash.com/X6Uj51n5CE8/150x150" class="img-thumbnail" alt="Profile Picture">
+    </div>
+    <div class="d-flex flex-column bd-highlight flex-grow-1 ml-2">
+      <div><h1><i>@<%= current_user.username %></i></h1></div>
+      <div><%= current_user.first_name %> <%= current_user.last_name %></div>
+      <div>📍 <%= current_user.location %></div>
+    </div>
+    <div>
+      <button type="button" class="btn btn-dark">Follow</button>
+    </div>
+  </div>
+
+ <div class="d-flex justify-content-center">
+    <p class="centered-content"><%= current_user.bio %></p>
+  </div>
+
+    <!--Grid row-->
```

Comment:
Here we need authorization, but this can wait for later, the the images also can't be static